docs/usage/webhook/deploy.rst

```diff
@@ -0,0 +1,124 @@
+.. _usage_ffmpeg_deploy:
+
+In this tutorial we will deploy a dataflow(ffmpeg dataflow) which converts a video to gif over an http server.We'll also
+see how to deploy the same http server in a docker container.Finally in :ref:`usage_ffmpeg_deploy_serve`
+we'll setup another http server which waits on gtihub webhooks to rebuilt and deploy the ffmpeg dataflow.
+
+Deploying on http server
+========================
+
+.. note::
+
+    All the code for this example is located under the
+    `examples/ffmpeg <https://github.com/intel/dffml/blob/master/examples/ffmpeg/>`_
+    directory of the DFFML source code.Rest of the tutorial is ran from the root of
+    this folder.
+
+Write operations and definitions to convert videos files to gif by calling
+``ffmpeg`` (Make sure you `download and install <https://www.ffmpeg.org/download.html>`_ it.).
+The operation accepts bytes (of the video), converts it to gif and outputs it as bytes.
+
+**ffmpeg/operations.py**
+
+.. literalinclude:: /../examples/ffmpeg/ffmpeg/operations.py
+
+Dataflow and Config files
+-------------------------
+
+**ffmpeg/dataflow.py**
```

Comment:
I don't see this file? Also, shouldn't this really be the part where we add the operation to `ffmpeg/setup.py` `entry_points`, re-install the package, and then use the `dffml dataflow create` command? Instead of `export`.

Reply:
Probably don't need the `dataflow.py` file anyway is what I'm saying

Reply:
sorry i forgot to commit that file.I added `resolution` as a seed input,so that we can use `input_mode:bytes` and just post the file.
which one would be better, current or sending `resolution` and `input_file` with `input_mode:default`?

Reply:
Keep resolution in the `seed`. We'll have to implement multipart form encoding to do that correctly with `default`